path/src/cmd/debug.ps1

```
@@ -0,0 +1,11 @@
+﻿function script:cmd_debug {
+	bootstrap_full @args
+	$args = @($args | Select-Object -Skip 1)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)